node/clients/masterdata.ts

```diff
@@ -1,30 +1,29 @@
-import type { InstanceOptions, IOContext } from '@vtex/api'
-import { JanusClient } from '@vtex/api'
+import type { InstanceOptions, IOContext } from "@vtex/api";
+import { ExternalClient } from "@vtex/api";
+import { DATA_ENTITY_NAME, FIELDS, SCHEMA } from "../utils/constant";
+import { Wishlist } from "../typings/wishlist";
 
-import { DATA_ENTITY_NAME, FIELDS, SCHEMA } from '../utils/constant'
-import { Wishlist } from '../typings/wishlist'
-
-export default class MasterDataClient extends JanusClient {
+export default class MasterDataClient extends ExternalClient {
   constructor(context: IOContext, options?: InstanceOptions) {
-    super(context, {
+    super(`https://${context.account}.vtexcommercestable.com.br`, context, {
       ...options,
       headers: {
         ...options?.headers,
-        VtexIdclientAutCookie: context.authToken,
-        Accept: 'application/vnd.vtex.pricing.v3+json',
-        'Cache-Control': 'no-cache',
-        'Content-Type': 'application/json',
+        ...{ Accept: "application/vnd.vtex.ds.v10+json" },
+        ...(context.authToken
+          ? { VtexIdclientAutCookie: context.authToken }
+          : null),
       },
-    })
+    });
   }
 
   public async getWishlist(id: string): Promise<Wishlist> {
     try {
```

Comment:
we can remove async await and try catch from the client. we don't need to handle errors and return.

Reply:
Done